training/general/src/environments/tensorflow-2.8-cuda11/context/conda_dependencies.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - azureml-mlflow=={{latest-pypi-version}}
   - azureml-telemetry=={{latest-pypi-version}}
   - tensorboard~=2.8.0
-  - tensorflow-gpu~=2.8.0
+  - tensorflow-gpu~=2.8.4
```

Comment:
~=2.8.4 is the same as ~=2.8.0 and would pick the highest patch version for tf28

Reply:
it wasn't picking up the latest patch because the protobuf version didn't work with 2.8.4. So I downgraded protobuf so I switched it back to 2.8.0.